pkg/asset/internal/templates.go

```diff
@@ -152,6 +152,8 @@ spec:
         - --runtime-config=api/all=true
         - --tls-cert-file=/etc/kubernetes/secrets/apiserver.crt
         - --tls-private-key-file=/etc/kubernetes/secrets/apiserver.key
+        - --kubelet-client-certificate=/etc/kubernetes/secrets/apiserver-client.crt
+        - --kubelet-client-key=/etc/kubernetes/secrets/apiserver-client.key
```

Comment:
The apiserver cert we have been generating has both client/server key usages - do we really want to separate them? Not necessarily opposed, but it is more moving parts, and I'm not sure how necessary it really is.